cvat/apps/engine/task.py

```diff
@@ -184,6 +189,30 @@ def _validate_data(counter, manifest_file=None):
 
     return counter, task_modes[0]
 
+def _copy_data_from_clowder(api_key, file_info, upload_dir):
+    from cvat.apps.engine.clowder_api import ClowderApi
+
+    job = rq.get_current_job()
+    local_files = {}
```

Comment:
Maybe better use `list` here?

Reply:
@Marishka17 Thank you! We changed it to `set`.